wolfcrypt/src/random.c

```diff
@@ -2741,14 +2745,26 @@ int wc_GenerateSeed(OS_Seed* os, byte* output, word32 sz)
         }
     #endif /* HAVE_INTEL_RDSEED */
 
+#ifdef WIN_REUSE_CRYPT_HANDLE
+    if (gHandle == 0) {
+        if(!CryptAcquireContext(&gHandle, 0, 0, PROV_RSA_FULL,
+                                CRYPT_VERIFYCONTEXT))
+            return WINCRYPT_E;
+    }
+    os->handle = gHandle;
```

Comment:
if `WIN_REUSE_CRYPT_HANDLE` is enabled `os->handle` seems to be unused and can be removed from `OS_Seed`

Reply:
Keeping for compatibility (structure needs a member)